### ISSUE TITLE
bug in latest flutter on master channel

### ISSUE THREAD

**Issue by DJafari** [johnpryan/flutter_map#364]
```
type '_BroadcastStream<void>' is not a subtype of type 'Stream<int>'

When the exception was thrown, this was the stack:
#0      PolygonLayer._build (package:flutter_map/src/layer/polygon_layer.dart:54:15)
#1      PolygonLayer.build.<anonymous closure> (package:flutter_map/src/layer/polygon_layer.dart:47:16)
```

[this line](https://github.com/johnpryan/flutter_map/blob/3229089b82ec6692d456da6ec9d5657a2706e1ea/lib/src/layer/polygon_layer.dart#L52)

**Comment by figux01** [comment]
I have the same issue in my project.

**Comment by johnpryan** [comment]
fixed in 0.7.0
